### PR TITLE
Version bump to 1.5.0

### PR DESCRIPTION
**Pull request by lukecarbis**
### 1.5.0 – 2019-10-28 ###

 Ready for a big release? We're really happy to be introducing quite a number of highly requested features, including a PHP API for registering blocks with code, a new text field with lists and headings, and some neat workflow efficiencies when building your block. 

 * New: There's now a PHP API for registering blocks using code (instead of the WP Admin UI). Documentation is [here](https://github.com/getblocklab/block-lab/pull/434) for now, but more on its way soon
 * New: Classic Text control (for Block Lab Pro users)! This field is similar to Rich Text, but has a few extra options for things like lists and headings
 * New: Duplicate fields – building your block is now so much easier, with the ability to duplicate rows
 * New: Repeater Row Count function – a helper function that returns the total amount of rows in a given repeater. Documentation [here](https://github.com/getblocklab/block-lab/pull/429)
 * New: Repeater Row Index function – a helper function that returns the current row, while looping through a repeater. Documentation [here](https://github.com/getblocklab/block-lab/pull/429)
 * Tweak: We've removed our dependency on global variables. This is mostly a best practice thing, not user facing. More details [here](https://github.com/getblocklab/block-lab/pull/435).
 * Tweak: We've refactored quite a lot about our block Loader class, to make it more robust, secure, and maintainable
 * Tweak: Loads of new unit and integration tests - these help prevent us from introducing bugs or regressions in the future
 * Fix: Bug which affected sites which had removed or renamed the admin user role
 * Fix: Empty Rich Text fields now no longer output a single `</p>` tag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Contributors: lukecarbis, ryankienstra, Stino11, rheinardkorf
 Tags: Gutenberg, Blocks
 Requires at least: 5.0
-Tested up to: 5.2
+Tested up to: 5.3
```
Comment (kienstra) [inline README.md]: I think it'd be good to fix #456 before bumping this.

Comment (kienstra) [reply README.md]: Also, the Multi-select looks strange in WP 5.3-RC2:

mult-select

**Comment by kienstra** [comment]
Hi @lukecarbis,
This looks good. It bumps the plugin version in the same files as [previous version bump PRs](https://github.com/getblocklab/block-lab/pull/413/files).

It looks like there are a few styling issues with WP 5.3-RC2, and there could be more with more testing.

I think it'd be good to wait until our next minor bump the 'Tested up to' to `5.3`. But not a strong opinion.

**Review by kienstra — APPROVED** [review]

**Comment by lukecarbis** [comment]
@kienstra Agreed! Done in https://github.com/getblocklab/block-lab/pull/455/commits/8774980f2ba0d3f4462a57d24ab95a61631042fa.

**Comment by kienstra** [comment]
Looks good!

**Pull request merged**